src/pages/test-report/cat-be/components/reverse-left-popover/reverse-left-popover.html

```diff
@@ -0,0 +1,21 @@
+<ng-container>
+  <ion-grid class="popover-content">
+    <ion-row>
+      <ion-col col-36>
+      </ion-col>
+      <ion-col class="manoeuvre-competencies-container">
+        <manoeuvre-competency 
+          [manoeuvre]="manoeuvreTypes.reverseLeft"
+          [competency]="competencies.controlFault"
+          [attr.id]="getId(competencies.controlFault)"
+        ></manoeuvre-competency>
+        <manoeuvre-competency
+          class="second"
+          [attr.id]="getId(competencies.observationFault)"
+          [manoeuvre]="manoeuvreTypes.reverseLeft"
+          [competency]="competencies.observationFault"
+          ></manoeuvre-competency>
```

Comment:
Could we have the attributes in the same order please, just makes it easier to compare. (trivial i know).
Also is the class second being used to apply styling/positioning?

Reply:
Changed the order and deleted that class